src/CartingService/Application/Cart/Commands/AddCartItem/AddItemCommand.cs

```diff
@@ -33,10 +33,13 @@ public Task<bool> Handle(AddItemCommand request, CancellationToken cancellationT
         var cart = _cartRepository.GetCart(request.CartId);
         if (cart == null)
         {
+            //todo how does exceptions are handled
             throw new CartServiceException("The cart does not exist");
         }
         
         var newItem = _mapper.Map<CartItem>(request.Item);
+        
+        //todo this looks like business logic, shouldn't it be in the Domain layer instead?
```

Comment:
Agree, In the first implementation I put it in the domain. But I do not remember the reason, why I moved here.